Content.Shared/_DV/CosmicCult/Components/Examine/CosmicMarkBlankComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.StatusIcon;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+
+namespace Content.Shared._DV.CosmicCult.Components.Examine;
```

Comment:
Remove unused using directives
```suggestion
using Robust.Shared.Prototypes;

namespace Content.Shared._DV.CosmicCult.Components.Examine;
```